sidekiq/metadata.csv

```diff
@@ -4,16 +4,19 @@ sidekiq.jobs.perform.avg,gauge,,,,Average amount of time spent in a worker,0,sid
 sidekiq.jobs.perform.max,gauge,,,,Max amount of time spent in a worker,0,sidekiq,jobs perform max
 sidekiq.jobs.perform.median,gauge,,,,Median amount of time spent in a worker,0,sidekiq,jobs perform median
 sidekiq.jobs.perform.95percentile,gauge,,,,95th percentile of amount of time spent in a worker,0,sidekiq,jobs perform perc
-sidekiq.jobs.count,count,,,,Count of Sidekiq jobs,0,sidekiq,jobs count
-sidekiq.jobs.success,count,,,,Count of successful Sidekiq jobs,0,sidekiq,jobs success
-sidekiq.jobs.failure,count,,,,Count of failed Sidekiq jobs,-1,sidekiq,jobs failure
-sidekiq.processed,count,,,,Number of job executions completed (success or failure),0,sidekiq,job execution processed
-sidekiq.failures,count,,,,Number of job executions which raised an error,-1,sidekiq,jobs execution failures
-sidekiq.enqueued,count,,,,Total size of all known queues,0,sidekiq,jobs enqueued
-sidekiq.retries,count,,,,Total retries size,0,sidekiq,job retries
-sidekiq.dead,count,,,,Total Dead Size,0,sidekiq,jobs dead
-sidekiq.scheduled,count,,,,Total Scheduled Size,0,sidekiq,jobs scheduled
-sidekiq.busy,count,,,,Total Busy Size,0,sidekiq,jobs busy
+sidekiq.jobs.count,count,,,,Total count of Sidekiq jobs,0,sidekiq,jobs count
+sidekiq.jobs.success,count,,,,Total count of successful Sidekiq jobs,0,sidekiq,jobs success
+sidekiq.jobs.failure,count,,,,Total count of failed Sidekiq jobs,-1,sidekiq,jobs failure
+sidekiq.jobs.worker.count,count,,,,Count of Sidekiq jobs,0,sidekiq,jobs count
+sidekiq.jobs.worker.success,count,,,,Count of successful Sidekiq jobs,0,sidekiq,jobs success
+sidekiq.jobs.worker.failure,count,,,,Count of failed Sidekiq jobs,-1,sidekiq,jobs failure
```

Comment:
It seems `unit_name` are missing. https://github.com/DataDog/dogweb/blob/prod/integration/system/units_catalog.csv

`sidekiq.jobs.count` should have `unit_name` = `job` for example.

Reply:
Sure!